tc-client/src/main/java/com/tc/client/ClientFactory.java

```diff
@@ -30,7 +30,7 @@ public class ClientFactory {
   //  we can verify that it won't be used here.
   public static DistributedObjectClient createClient(Iterable<InetSocketAddress> serverAddresses, ClientBuilder builder,
                                                      TCThreadGroup threadGroup,
-                                                     String uuid, String name, boolean async) {
-    return new DistributedObjectClient(serverAddresses, builder, threadGroup, uuid, name, async);
+                                                     String uuid, String name) {
+    return new DistributedObjectClient(serverAddresses, builder, threadGroup, uuid, name);
```

Comment:
I'd leave this with the BasicConnectionManager for diagnostic connections.  Basic has been stable for a while and I think synchronous is way better for those cases.